### PR TITLE
doc/orchestrator: Fix various issues in Orchestrator CLI documentation

### PR DESCRIPTION
**Pull request by votdev**
Signed-off-by: Volker Theile <vtheile@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by votdev** [comment]
@sebastian-philipp Could you please add some more reviewers, please.

**Comment by sebastian-philipp** [comment]
> @sebastian-philipp Could you please add some more reviewers, please.

no need for that. Let's merge this as soon as the tests pass.

File: doc/mgr/orchestrator_cli.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Orchestrator CLI
 ================
```
Comment (sebastian-philipp) [inline doc/mgr/orchestrator_cli.rst]: could you prefix your commit with `doc/orchestrator` please?

Comment (votdev) [reply doc/mgr/orchestrator_cli.rst]: Done

**Pull request merged**